rpc/client/client.go

```diff
@@ -325,8 +325,12 @@ func (c *Client) GenesisChunked(context context.Context, id uint) (*ctypes.Resul
 func (c *Client) BlockchainInfo(ctx context.Context, minHeight, maxHeight int64) (*ctypes.ResultBlockchainInfo, error) {
 	const limit int64 = 20
 
-	minHeight, maxHeight, err := filterMinMax(
-		int64(c.node.BlockManager.State.BaseHeight),
+	baseHeight, err := c.node.BlockManager.Store.LoadBaseHeight()
```

Comment:
currently we only set base height when pruning. it means if we never set pruning, will never have base height and we won't be able to serve this method.

when was base height saved before?